themes/bootstrap3/scss/vendor/bootstrap/_theme.scss

```diff
@@ -247,6 +247,7 @@
   border-radius: $border-radius-base;
   @include box-shadow(0 1px 2px rgba(0, 0, 0, .075));
 }
+
```

Comment:
Any idea how/why this blank line got introduced here?

Reply:
Sorry, my fault. Erroneously had edited the wrong file

Reply:
I went ahead and reverted the change to simplify the PR.